### ISSUE TITLE
Cannot place plaster to desired rotation.

### ISSUE THREAD

**Issue by fipil** [anegostudios/VintageStory-Issues#758]
**Game Version:** 1.14.5
**Platform:** Client on Windows, server on linux
**Modded:** Only client mod HudClock.

### Description
Hi, there is not possible to place a plaster with diagonal logs as needed - to be properly rotated.
The wiki contains this only sentence describing howto place it: "Special: When placed, the diagonal and straight plaster blocks present different faces in different directions. The block facing is dependent on the direction that a player faces when placing the block."
The description is not accurate, because as soon as I found by experimenting with that, it depens on the face of the block to which user clicks by the right mouse when holding the plaster block. In this time player faces towards the block, but player itself can stand on many places and can be rotated many ways.

The following screenshot shows how I need to place the plaster.
![2021-01-16_10-15-09_LI](https://user-images.githubusercontent.com/768590/104808227-0cdefe00-57e5-11eb-855a-1d0a4728aaf9.jpg)

The following screenshot shows a real usage and a reason why I need to place it properly - so the diagonal logs are conecting each other.
![2021-01-16_10-13-07_LI](https://user-images.githubusercontent.com/768590/104808254-30a24400-57e5-11eb-992a-82a45651c0f6.jpg)

### How to reproduce
Try to place it as shown in my screenshot.

### Expected behavior
A possibility to place plaster rotated in any desired rotation around any of the three axes.

The inability to rotate a block is only pain I have in VintageStory. It would be nice if there is some tool, such is wrench in MC, allowing us rotating any block to any angle. It would solve a lot of those problems. Also if I make a multimaterial chiseled block and use log for example, the block is rotated forever the same direction as in time of chiselling. So I have to chisel only at the target place or knowing where I'd like to place.
The same problems are with chutes, such I reported in another issue.

The VintageStory is a great game, I love and enjoy it so much. Fixing of this problem would be highly appreciated.
Keep up your perfect work!
Filip

**Comment by fipil** [comment]
An additional note: I understand that a Wrench is probably not so medieval tool. The problem is not in tool itself, the problem is in controlling game. It's on the same level such is placing blocks, I simply cannot place it as I want. So maybe it shouldn't be a tool, it should be a (configurable) keystroke?

**Comment by copygirl** [comment]
I'll mark this as an enhancement because from what I know it is possible to rotate the diagonal plaster blocks in all the possible orientations, but it can be really annoying. Sometimes you need to look up or down to place them the way you want.

**Comment by copygirl** [comment]
Disregard what I just said. These plaster blocks really do only have 3 possible orientations, which doesn't fit all the possible ways someone might want to lay them out.

![image](https://user-images.githubusercontent.com/272586/104811306-c85e5d00-57fa-11eb-8375-3b589c842e64.png)

The three blocks closest to the camera are the three orientations. The ones facing away are just the same as those opposite of them. Ideally, we'd have 3 additional orientations where the diagonal logs are rotated but the plain side stays where it is.

**Comment by Craluminum2413** [comment]
No longer relevant since 1.17 that added wrench tool

**Comment by copygirl** [comment]
@Craluminum2413 The other important question is whether plaster actually has 6 orientations now. So far I think it acted like a log block.

**Comment by Craluminum2413** [comment]
You are right, I tried to rotate plaster with a wrench tool and there is no such variant

**Comment by RedramVS** [comment]
I don't think this issue can be perfectly solved by 3 more variants.  Unless each face is individually rotate-able, there will always be the possibility of it not being able to meet the specific configuration the player wants.

**Comment by Craluminum2413** [comment]
@RedramVS Will it gets fixed someday?
